packages/web-runtime/src/App.vue

```diff
@@ -162,12 +103,6 @@ export default defineComponent({
     )
   },
   methods: {
-    focusModal(component, event) {
-      ;(this as any).focus({
-        revert: event === 'beforeUnmount'
-      })
-    },
-
```

Comment:
Is this now handled anywhere else?

Reply:
No, but I didn't find a reason to keep it. Modals are being focused correctly due to the focus trap in `OcModal`.